app/views/reviews/_form.html.erb

```diff
@@ -42,6 +42,11 @@
 
   </div>
 
+  <div class="field">
+    <%= form.label :maxDevTime %>
+    <%= form.select(:maxDevTime, [[0,0],[15,15] ,[30,30], [60,60]])%>
```

Comment:
You should probably get the values from the server and not determine them on the client, I'd use `enumify` abilities here